### PR TITLE
fix: changes in environment files

### PR DESCRIPTION
**Pull request by Hardeep8587**
## Change Description

changes in environment files

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist: (Check only applicable)

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/environments/environment.prod.ts [hunk 1]
```diff
@@ -9,6 +9,8 @@ export const environment = {
   isAccountsEnabled: false,
   BaseUrl: 'https://service.cloud.fleethawks.com/api/v1/',
   AssetURL: 'https://fh-cloud-service-uploads-ue1.s3.us-east-1.amazonaws.com',
+  AccountServiceUrl: 'http://localhost:4002/api/v1/',
+  SafetyUrl: 'http://localhost:4000/api/v1/',
```
Comment (kunalfleethawks) [inline src/environments/environment.prod.ts]: Change this to SafetyServiceUrl

File: src/environments/environment.ts [hunk 2]
```diff
@@ -6,9 +6,10 @@ export const environment = {
   production: false,
   isFeatureEnabled: true,
 
-  BaseUrl: 'https://fleetservice.ap-south-1.fleethawks.com/api/v1/',
+   BaseUrl: 'https://fleetservice.ap-south-1.fleethawks.com/api/v1/',
  // BaseUrl: 'http://localhost:3000/api/v1/',
-  AccountServiceUrl: 'http://localhost:4002/api/v1/',
+ // AccountServiceUrl: 'http://localhost:4002/api/v1/',
```
Comment (kunalfleethawks) [inline src/environments/environment.ts]: Uncomment this

File: src/environments/environment.ts [hunk 3]
```diff
@@ -6,9 +6,10 @@ export const environment = {
...
+ // SafetyUrl: 'http://localhost:4000/api/v1/',
```
Comment (kunalfleethawks) [inline src/environments/environment.ts]: Uncomment this and change it to SafetyServiceUrl

**Review by kunalfleethawks — CHANGES_REQUESTED** [review]
@Hardeep8587 Please address the comments

File: src/environments/environment.ts (hunk 3 above)
Comment (Hardeep8587) [reply src/environments/environment.ts]: done

**Comment by Hardeep8587** [comment]
@kunalfleethawks please review

**Pull request merged**